### PR TITLE
Feature/il 5073 Morpho rewards

### PR DESCRIPTION
**Pull request by pete-ipor**

### CODE REVIEW

File: contracts/rewards_fuses/morpho/MorphoClaimFuse.sol [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+     * @param version Address of this contract instance
+     * @param rewardsToken Address of the token being claimed as rewards
+     * @param rewardsTokenAmount Amount of rewards tokens claimed
+     * @param rewardsClaimManager Address of the RewardsClaimManager receiving the claimed rewards
+     */
+    event MorphoClaimFuseRewardsClaimed(
+        address version,
+        address rewardsToken,
+        uint256 rewardsTokenAmount,
+        address rewardsClaimManager
```
Comment (mario-ipor) [inline contracts/rewards_fuses/morpho/MorphoClaimFuse.sol:30]: why rewardsClaimManager is a part of event?

File: contracts/rewards_fuses/morpho/MorphoClaimFuse.sol [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    );
+
+    /**
+     * @notice Thrown when the RewardsClaimManager address is zero
+     * @param version Address of this contract instance
+     */
+    error MorphoClaimFuseRewardsClaimManagerZeroAddress(address version);
```
Comment (mario-ipor) [inline contracts/rewards_fuses/morpho/MorphoClaimFuse.sol:37]: We already have error for that Errors.WrongAddress

File: contracts/rewards_fuses/morpho/MorphoClaimFuse.sol [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+    /**
+     * @notice Thrown when the UniversalRewardsDistributor address is zero
+     * @param version Address of this contract instance
+     */
+    error MorphoClaimFuseUniversalRewardsDistributorZeroAddress(address version);
```
Comment (mario-ipor) [inline contracts/rewards_fuses/morpho/MorphoClaimFuse.sol:43]: We already have error for that Errors.WrongAddress

**Review by mario-ipor — APPROVED** [review]

File: contracts/rewards_fuses/morpho/MorphoClaimFuse.sol (hunk 1 above)
Comment (rzonsol) [reply contracts/rewards_fuses/morpho/MorphoClaimFuse.sol:30]: Since I wanted to know where the funds were delivered after the event.

**Pull request merged**